control/control.SMO.xml

```diff
@@ -513,7 +513,7 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
                 </proposal_module>
                 <!-- FaTE #303859 - simple network (in fact firewall) cfg in 1st stage -->
                 <proposal_module>
-                    <name>firewall</name>
+                    <name>security</name>
```

Comment:
here it is wrong. It has to be before software above, otherwise proposing selinux pattern won't work.

Reply:
Updated the order